### PR TITLE
Introduce a new provider: Microsoft Azure Stack

### PR DESCRIPTION
**Pull request by gberginc**
With this post we are introducing a new provider for ManageIQ: Microsoft Azure Stack. 
The post first explains a bit of a background and then guides a user through the
most common functionalities supported by the current version of the
provider.

### CODE REVIEW

File: site/_posts/2019-12-06-introducing-new-provider-microsoft-azure-stack.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Introducing New Cloud Provider: Microsoft Azure Stack"
+author: XLAB Steampunk
+date: 2019-12-06 8:00:00 UTC
+comments: true
+published: true
+tags: provider, microsoft, azure stack, collaboration
```
Comment (cybette) [inline site/_posts/2019-12-06-introducing-new-provider-microsoft-azure-stack.md]: Please use existing tags, may I suggest: announcements, collaboration

File: site/_posts/2019-12-06-introducing-new-provider-microsoft-azure-stack.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Introducing New Cloud Provider: Microsoft Azure Stack"
+author: XLAB Steampunk
+date: 2019-12-06 8:00:00 UTC
```
Comment (cybette) [inline site/_posts/2019-12-06-introducing-new-provider-microsoft-azure-stack.md]: If you like you can update the date to today (optional)

**Comment by cybette** [comment]
Looks great, thanks @gberginc! I can merge after you've made the minor changes.

**Comment by gberginc** [comment]
@cybette thanks for the review. I have updated the tags according to your proposal and set the date to tomorrow, Tuesday, December 10, 2019.

**Pull request merged**